src/EditorFeatures/Text/Microsoft.CodeAnalysis.EditorFeatures.Text.csproj

```diff
@@ -30,12 +30,9 @@
     <InternalsVisibleTo Include="Microsoft.CodeAnalysis.EditorFeatures" />
     <InternalsVisibleTo Include="Microsoft.CodeAnalysis.EditorFeatures.Wpf" />
     <InternalsVisibleTo Include="Microsoft.CodeAnalysis.VisualBasic.EditorFeatures" />
-<<<<<<< HEAD
     <InternalsVisibleTo Include="Microsoft.VisualStudio.CSharp.Repl" />
     <InternalsVisibleTo Include="Microsoft.VisualStudio.InteractiveServices" WorkItem="https://github.com/dotnet/roslyn/issues/35098" />
```

Comment:
@RikkiGibson I don't know about the other lines, but this one should have been removed.